src/dsp_tools/commands/excel2json/models/json_header.py

```diff
@@ -9,13 +9,13 @@
 
 @dataclass
 class JsonHeader(Protocol):
-    def make(self) -> dict[str, Any]:
+    def serialise(self) -> dict[str, Any]:
```

Comment:
technically, serialisation is the process of turning objects into strings, so a function that returns a dict is not really a function that serialises. I'm ok with it here because python dicts are so close to JSON - but it is a bit imprecise.

Reply:
Ok, thanks. What would you say is a better name?